crates/re_space_view/src/space_view_contents.rs

```diff
@@ -95,8 +95,7 @@ pub struct DataBlueprintTree {
     /// Two things to keep in sync:
     /// * children on [`DataBlueprintGroup`] this is on
     /// * elements in [`Self::path_to_group`]
-    /// TODO(andreas): Can we reduce the amount of these dependencies?
-    entity_paths: IntSet<EntityPath>,
+    per_system_entity_list: PerSystemEntities,
```

Comment:
This should be skip serde for now (until it's totally refactored anyways when we stop using serde-field, etc.)